Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Subtitle/GetById/GetSubtitleByIdHandler.cs

```diff
@@ -2,9 +2,10 @@
 using FluentResults;
 using MediatR;
 using Streetcode.BLL.DTO.AdditionalContent.Subtitles;
+using Streetcode.BLL.MediatR.AdditionalContent.GetById;
 using Streetcode.DAL.Repositories.Interfaces.Base;
 
-namespace Streetcode.BLL.MediatR.Subtitle.GetById;
+namespace Streetcode.BLL.MediatR.AdditionalContent.Subtitle.GetById;
```

Comment:
In the most recent commit namespaces in some of the files appear to be misleading. Please thoroughly review all newly created files and fix the latter.